charts/stable/gotify/values.yaml

```
@@ -0,0 +1,91 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/master/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: gotify/server
+  # -- image tag
+  tag: 2.1.4
```

```suggestion
  # -- image tag
  # @default -- chart.appVersion
  tag:
```
This makes it so that we only have to update the appVersion (by default) but it still allows users to override it if they want.